services/movie_session.py

```diff
@@ -0,0 +1,60 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+from tests.test_main import movie_session_database_data
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movie_to_create = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall,
+        movie=movie
+    )
+    return movie_to_create
+
+
+def get_movies_sessions(
+        session_date: str = None
+) -> QuerySet[MovieSession]:
+    queryset = MovieSession.objects.all()
+
+    if session_date:
+        queryset = queryset.filter(show_time__in=session_date)
```

Comment:
The filter condition `show_time__in=session_date` is incorrect. It should be `show_time__date=session_date` to filter sessions by date. Ensure `session_date` is in the correct format (e.g., 'YYYY-MM-DD').